modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.css

```
@@ -0,0 +1,16 @@
+.evp_news_container {
+	border: 1px solid;
+	border-radius: 8px;
+}
+
+.evp_news_container img {
+	height: 24px;
+	width: 24px;
+}
+
+.evp_news_container .news-content .news-text {
```

Please, change the class name here.
```suggestion
.evp-news-container .news-content .news-text {
```